doc/roadmap_2023.md

```diff
@@ -0,0 +1,186 @@
+# Roadmap for 2023
+
+2022 was an interesting year for Vello. We made good progress on a number of fronts, solving some really tricky problems, and also did a rewrite into WGSL, but did not ship a usable renderer. Oh, and we renamed the project from piet-gpu as well.
+
+2023 promises to be even more interesting. With some focus we can ship a usable renderer. The core team (Raph Levien and Chad Brokaw, with help from Arman Uguray) will be able to do most of the tasks listed here. Help from the community could speed things up or add even more features. We also rely heavily on the wgpu community to build solid GPU infrastructure for our uses, and also expect a nontrivial amount of integration work as people adopt Vello for their projects.
+
+Large parts of the [piet-gpu vision] are still valid, but of course there are updates in the two years since it was written.
+
+This roadmap is in roughly chronological in order of when we expect to complete the work.
+
+## Completing a basic imaging model
+
+A 2D renderer needs to support at least a basic imaging model. The biggest single thing missing from Vello right now is images; without those, it is unsuitable for many applications.
+
+Supporting images *well* is tricky, in large part because of limitations in GPU infrastructure. The number of images that may appear in a scene is not bounded, which is not a good fit for the basic descriptor binding model. Ideally a single shader (the fine rasterization stage) can sample from all the images in the scene directly, but that's not really possible in WebGPU 1.0. Perhaps a future extension will have version of this; in Vulkan it's descriptor indexing (and [buffer device address] and [descriptor buffer], as GPU approaches to this problem keep evolving, but it's less likely the latter will be standardized in WebGPU, as they're basically variants of raw pointers and thus extremely difficult to make sage).
```

Comment:
Just chiming in with a tiny typo fix. Though I suppose raw pointers are hard to make sage as well.
```suggestion
Supporting images *well* is tricky, in large part because of limitations in GPU infrastructure. The number of images that may appear in a scene is not bounded, which is not a good fit for the basic descriptor binding model. Ideally a single shader (the fine rasterization stage) can sample from all the images in the scene directly, but that's not really possible in WebGPU 1.0. Perhaps a future extension will have version of this; in Vulkan it's descriptor indexing (and [buffer device address] and [descriptor buffer], as GPU approaches to this problem keep evolving, but it's less likely the latter will be standardized in WebGPU, as they're basically variants of raw pointers and thus extremely difficult to make safe).
```